### DISCUSSION TITLE
How to use template to move daily notes yyyy-MM-dd.md to folder yyyy-MM?

### DISCUSSION THREAD

**Discussion by francisHugo** [SilentVoid13/Templater#1274] [Help] [answered]
Hi all,

I use yyyy-MM-dd as my daily notes' name, now I would like to use template to move it to the folder yyyy/MM.

Is there a method to do this?

BR

Francis

**Comment by Zachatoo** [comment, marked as answer]
The daily notes plugin supports the folder delimiter character. Instead of having `yyyy-MM-dd` as your daily note format, set it to `yyyy/MM/yyyy-MM-dd`, and it will create the folder and place the note in that folder for you whenever you create a daily note.

**Reply by francisHugo** [reply]
Thanks! @Zachatoo,   that's a perfect way!
